Sources/Instrumentation/MultiplexInstrument.swift

```diff
@@ -27,6 +27,12 @@ public struct MultiplexInstrument {
     }
 }
 
+extension MultiplexInstrument {
+    func firstInstance<I>(of instrument: I.Type) -> I? {
+        self.instruments.first(where: { $0 is I }) as? I
```

Comment:
Random sidenote: I REALLY miss Scala's `collect`:

It would have been `self.instruments.iterator.collect { case instrument: I => instrument }.first`

Maybe we'll get nicer partial functions some day... :)